packages/mgt-element/src/providers/Providers.ts

```diff
@@ -100,18 +99,17 @@ export class Providers {
    * @static
    * @memberof Providers
    */
-  public static me(): Promise<User> {
+  public static async me(): Promise<User> {
     if (!this.client) {
-      this._mePromise = null;
+      this._me = null;
       return null;
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Can we confirm that this linting comment should go away?

Reply:
nvm, it's not a promise.

Reply:
There is no longer a misused promise. My linter is not showing an error, is yours?

Reply:
nope.